### PR TITLE
Update kyber query

### PR DESCRIPTION
**Pull request by tiendv89**
I've checked that:

* [x] the query produces the intended results
* [x] the folder name matches the schema name
* [x] the schema name exists in Dune
* [x] views are prefixed with `view_`, functions with `fn_`.
* [x] the filename matches the defined view, table or function and ends with .sql
* [x] each file has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`

### CODE REVIEW

**Review by antonio-mendes — APPROVED** [review]
@masquot , I've reviewed and added a few missing UNION statements and was then able to run the SQL and get a result.

If possible, can you take a look and if it's all good, merge and deploy this PR please? 🙏  thanks in advance!

File: binance_smart_chain/dex/trades/insert_kyber.sql [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            "srcToken" token_a_address,
+            "dstToken" token_b_address,
+            contract_address AS exchange_contract_address,
+            evt_tx_hash AS tx_hash,
+            NULL::integer[] AS trace_address,
+            evt_index AS evt_index
+        FROM kyber."AggregationRouterV2_evt_Swapped"
+        WHERE evt_block_time >= start_ts AND evt_block_time < end_ts
+    ) dexs
+    INNER JOIN ethereum.transactions tx
```
Comment (antonio-mendes) [inline binance_smart_chain/dex/trades/insert_kyber.sql:101]: This should be joining on bsc.transactions

File: binance_smart_chain/dex/trades/insert_kyber.sql [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+            t.contract_address AS exchange_contract_address,
+            t.evt_tx_hash AS tx_hash,
+            NULL::integer[] AS trace_address,
+            t.evt_index
+        FROM
+            kyber."DMMPool_evt_Swap" t
+        INNER JOIN kyber."Kyber Swap: Factory_evt_PoolCreated" f ON f.pool = t.contract_address        
+        AND t.evt_block_time >= start_ts AND t.evt_block_time < end_ts
+
+        -- UNION ALL
```
Comment (antonio-mendes) [inline binance_smart_chain/dex/trades/insert_kyber.sql:79]: Uncomment here

File: binance_smart_chain/dex/trades/insert_kyber.sql [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+        FROM kyber."AggregationRouterV2_evt_Swapped"
+        WHERE evt_block_time >= start_ts AND evt_block_time < end_ts
+    ) dexs
+    INNER JOIN ethereum.transactions tx
+        ON dexs.tx_hash = tx.hash
+        AND tx.block_time >= start_ts
+        AND tx.block_time < end_ts
+        AND tx.block_number >= start_block
+        AND tx.block_number < end_block
+    LEFT JOIN erc20.tokens erc20a ON erc20a.contract_address = dexs.token_a_address
```
Comment (antonio-mendes) [inline binance_smart_chain/dex/trades/insert_kyber.sql:107]: The `erc20.tokens` abstractions does not currently exists on BSC on Dune.

File: polygon/dex/trades/insert_kyber.sql [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+            "srcToken" token_a_address,
+            "dstToken" token_b_address,
+            contract_address AS exchange_contract_address,
+            evt_tx_hash AS tx_hash,
+            NULL::integer[] AS trace_address,
+            evt_index AS evt_index
+        FROM kyber."AggregationRouterV2_evt_Swapped"
+        WHERE evt_block_time >= start_ts AND evt_block_time < end_ts
+    ) dexs
+    INNER JOIN ethereum.transactions tx
```
Comment (antonio-mendes) [inline polygon/dex/trades/insert_kyber.sql:102]: Same as before, the blockchain here should be Polygon

**Review by antonio-mendes — CHANGES_REQUESTED** [review]
👋  @tiendv89 ,

I left a few comments on things that need changing. Mainly problems on BSC.
Otherwise I've tested the Polygon (with a manual fix, but please push a commit with the fix I asked in the comments here) and Ethereum query and those work and return the expected result.


Let me know if you need any help!

**Review by antonio-mendes — APPROVED** [review]
Perfect! Thank you @tiendv89 !

@masquot , it's good for deployment 🧰

**Review by masquot — APPROVED** [review]
Thank you @tiendv89

**Pull request merged**